catalog-rest-service/src/main/java/org/openmetadata/catalog/security/JwtFilter.java

```diff
@@ -119,4 +135,21 @@ protected static String extractToken(MultivaluedMap<String, String> headers) {
     return source;
   }
 
+  protected static RSAPublicKey getParsedPublicKey() {
+    // public key in PEM format without content '---PUBLIC KEY---' and '---END PUBLIC KEY---'
+    String pubKey = oktaPublicKey;
+
+    // removes white spaces
+    String publicKey = pubKey.replace(" ", "");
+    try {
+      byte[] encode =  Base64.decodeBase64(publicKey);
+      X509EncodedKeySpec keySpecX509 = new X509EncodedKeySpec(encode);
+      KeyFactory keyFactory = KeyFactory.getInstance("RSA");
+      return (RSAPublicKey) keyFactory.generatePublic(keySpecX509);
+    } catch (NoSuchAlgorithmException | InvalidKeySpecException e) {
+      e.printStackTrace();
```

Comment:
never print stack trace. User Log.error check other places

Reply:
Addressed in the latest commit.